simapp/app.go

```diff
@@ -324,7 +324,7 @@ func NewSimApp(
 		vesting.NewAppModule(app.AccountKeeper, app.BankKeeper),
 		bank.NewAppModule(appCodec, app.BankKeeper, app.AccountKeeper),
 		capability.NewAppModule(appCodec, *app.CapabilityKeeper),
-		crisis.NewAppModule(&app.CrisisKeeper),
+		crisis.NewAppModule(&app.CrisisKeeper, true),
```

Comment:
Ideally we need to be check if crisis invariants pass based on flag passes to start and not a compile time Boolean

Reply:
OK, no problem. I asked about it on the issue page. Here, this is a simapp, not gaia.